tests/Maker/MakeRegistrationFormTest.php

```diff
@@ -70,5 +74,55 @@ public function getTestDetails()
             // registration_form_entity_guard_authenticate for details
             ->addPostMakeCommand('php bin/console cache:clear --env=test'),
         ];
+
+        yield 'registration_form_with_email_verification' => [MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeRegistrationForm::class),
+            [
+                'y', // no verify user
+                'jr@rushlow.dev', // from email address
+                'SymfonyCasts', // From Name
+                'n', // add UniqueEntity
+                'n', // no authenticate after
+                0, // route number to redirect to
+            ])
+            ->setRequiredPhpVersion(70200)
+            ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeRegistrationFormVerifyEmail')
+            ->addExtraDependencies('symfonycasts/verify-email-bundle')
+            ->assert(
+                function (string $output, string $directory) {
+                    $this->assertStringContainsString('Success', $output);
+
+                    $fs = new Filesystem();
+
+                    $generatedFiles = [
+                        'src/Security/VerifyEmail.php',
+                        'templates/registration/confirmation_email.html.twig',
+                    ];
+
+                    foreach ($generatedFiles as $file) {
+                        $this->assertTrue($fs->exists(sprintf('%s/%s', $directory, $file)));
+                    }
+                }
+            ),
+        ];
+
+        yield 'verify_email_functional_test' => [MakerTestDetails::createTest(
+            $this->getMakerInstance(MakeRegistrationForm::class),
+            [
+                'y', // no verify user
+                'jr@rushlow.dev', // from email address
+                'SymfonyCasts', // From Name
+                'n', // add UniqueEntity
+                '', // yes authenticate after
+                'app_register',
+            ])
+            ->setRequiredPhpVersion(70200)
+            ->setFixtureFilesPath(__DIR__.'/../fixtures/MakeRegistrationFormVerifyEmailFunctionalTest')
+            ->addExtraDependencies('symfonycasts/verify-email-bundle')
+            ->configureDatabase()
+            ->updateSchemaAfterCommand()
+            ->addExtraDependencies('mailer')
+            ->addExtraDependencies('twig'),
```

Comment:
Oh, of course! Two things:

1) TwigBundle is already listed as a dependency of `MakeRegistrationForm`, so it shouldn't (I believe) be needed here?

2) If you choose "yes" to email verification, we should include "mailer" in the message about installing symfonycasts/verify-email-bundle IF mailer is not installed.

Reply:
done